README.md

```diff
@@ -28,6 +28,6 @@ cd ic101
 dfx start --background
 dfx canister create --all
 dfx build
-dfx canister install --all
+dfx canister install --all [--mode='reinstall']
```

Comment:
```suggestion
dfx canister install --all --mode=reinstall
```

No need for optionality here, reinstall also works if the canister isn't installed yet.